test/smoke/usm-globals-with-pragma/Makefile

```diff
@@ -0,0 +1,21 @@
+include ../../Makefile.defs
+
+TESTNAME     = usm_globals_with_pragma
+TESTSRC_MAIN = usm_globals_with_pragma.cpp
+TESTSRC_AUX  =
+TESTSRC_ALL  = $(TESTSRC_MAIN) $(TESTSRC_AUX)
+RUNENV      += HSA_XNACK=1 LIBOMPTARGET_KERNEL_TRACE=2
+
+RUNCMD       = ./$(TESTNAME) 2>&1 | $(AOMP)/bin/FileCheck $(TESTSRC_MAIN)
+
+CFLAGS       += -O3
+CLANG        = clang++
+OMP_BIN      = $(AOMP)/bin/$(CLANG)
+CC           = $(OMP_BIN) $(VERBOSE)
+
+SUPPORTED    = gfx90a,gfx940,gfx941,gfx942
```

Comment:
Is it possible to use wildcards (e.g., gfx94*). So you don't have to touch every single Makefile if the gfx number is incremented again.